types/convert.go

```diff
@@ -37,34 +37,64 @@ func truncateStr(str string, flen int) string {
 	return str
 }
 
-// UnsignedUpperBound indicates the max uint64 values of different mysql types.
-var UnsignedUpperBound = map[byte]uint64{
-	mysql.TypeTiny:     math.MaxUint8,
-	mysql.TypeShort:    math.MaxUint16,
-	mysql.TypeInt24:    mysql.MaxUint24,
-	mysql.TypeLong:     math.MaxUint32,
-	mysql.TypeLonglong: math.MaxUint64,
-	mysql.TypeBit:      math.MaxUint64,
-	mysql.TypeEnum:     math.MaxUint64,
-	mysql.TypeSet:      math.MaxUint64,
+// ByteToUnsignedUpperBound indicates the max uint64 values of different mysql types.
+func ByteToUnsignedUpperBound(b byte) uint64 {
```

Comment:
The parameter is not general byte, its integer type.
I think the name should be `IntergerUnsignedUpperBound(intType byte)`.

Reply:
@coocood May I ask why you say it is an `integer type`?

I basically replaced `var UnsignedUpperBound = map[byte]uint64{` to this function. And `UnsignedUpperBound` is a map with key as `byte`, shouldn't I use `byte` as type here as well? 🤔

Reply:
Oh, I see you are talking bout the name not the type... Aha. Sorry I miss read your input. I will use `intType` as the name then :)